pkg/langdetect/langdetect.go

```diff
@@ -61,12 +60,10 @@ func FindProjectType() (ProjectType, *ProjectDetails, error) {
 
 		case "Cargo.lock":
 			return RustCargo, &ProjectDetails{}, nil
-		
+
 		case "package-lock.json":
 			return JsNpm, &ProjectDetails{}, nil
 
-		default:
```

Comment:
Will bsf now work in an empty directory?

Reply:
yep